var/spack/repos/builtin/packages/py-cython/package.py

```diff
@@ -15,6 +15,7 @@ class PyCython(PythonPackage):
 
     license("Apache-2.0")
 
+    version("3.0.11", sha256="7146dd2af8682b4ca61331851e6aebce9fe5158e75300343f80c07ca80b1faff")
```

Comment:
I can only confirm the sha256 if I add the `url` property since they case of the first letter of the archive file in the `pypi` property doesn't match that of the new version.

```suggestion
    version("3.0.11", sha256="7146dd2af8682b4ca61331851e6aebce9fe5158e75300343f80c07ca80b1faff")
    version(
        "3.0.11",
        sha256="7146dd2af8682b4ca61331851e6aebce9fe5158e75300343f80c07ca80b1faff",
        url=f"https://files.pythonhosted.org/packages/source/cython/cython-3.0.11.tar.gz",
    )
```

@adamjstewart Do you have any suggestions about a better way to handle this issue?